### ISSUE TITLE
.babelrc and runtime error `babelHelpers.asyncToGenerator is not a function`

### ISSUE THREAD

**Issue by lettertwo** [matc4/react-native-svg-uri#62]
tl;dr: I think the .babelrc needs to be excluded from the npm module.

I've read the current '[best practice](https://github.com/facebook/react-native/issues/10966#issue-189613897)' is to ship invalid node modules when they are for RN, but because of [how babelrc lookup works](https://babeljs.io/docs/usage/babelrc/#lookup-behavior), the presence of a `.babelrc` in this package creates a conflict with React Native's packager's babel config and causes weird runtime errors like facebook/react-native#4844 ([this comment](https://github.com/facebook/react-native/issues/4844#issuecomment-204035720) is enlightening in particular).
